main/platformio.ini

```diff
@@ -8,11 +8,18 @@
 ; Please visit documentation for the other options and examples
 ; https://docs.platformio.org/page/projectconf.html
 
-[env:ttgo-t-beam]
+; [env:ttgo-t-beam]
+; platform = espressif32
+; board = ttgo-t-beam
+; framework = arduino
+; monitor_speed = 115200
+
+[env:lolin32_lite]
 platform = espressif32
-board = ttgo-t-beam
+board = lolin32_lite
 framework = arduino
-monitor_speed = 115200
+monitor_speed=115200
 
 lib_deps = mikalhart/TinyGPSPlus@^1.1.0
-           adafruit/Adafruit BMP280 Library@^2.6.8
\ No newline at end of file
+           adafruit/Adafruit BMP280 Library@^2.6.8
+
```

Comment:
Could you maybe try to uncomment everything here - I mean keep two configurations, and try if you can run the program?

Platformio might be able to detect the correct one by itself

Reply:
It can't detect it automatically as far as I tested it but you can just manually set the config once in platformio GUI (at the bottom panel) and it will build only this one config